src/current/cockroachcloud/managed-backups.md

```diff
@@ -0,0 +1,382 @@
+---
+title: Managed Backups
+summary: Learn about CockroachDB Cloud managed backups.
+toc: true
+docs_area: manage
+---
+
+{% include cockroachcloud/backups/managed-backup-description.md %}
+
+You can configure the following in [{{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters](#standard-and-advanced-clusters):
+
+- The frequency of the backups to meet [recovery point objective (RPO)]({% link {{site.current_cloud_version}}/disaster-recovery-overview.md %}#resilience-strategy) requirements.
+- The retention of the backups to set how long Cockroach Labs retains the backups.
+
+In [{{ site.data.products.basic }} clusters](#basic-clusters), you can view the default daily managed backups in the Cloud Console.
+
+{{site.data.alerts.callout_info}}
+In addition to managed backups, you can take manual backups to your own storage bucket with self-managed backups. Refer to the [Take and Restore Self-Managed Backups]({% link cockroachcloud/take-and-restore-self-managed-backups.md %}) page.
+{{site.data.alerts.end}}
+
+## Managed backup settings
+
+You can modify the settings of managed backups for [{{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters](#standard-and-advanced-clusters).
+
+[{{ site.data.products.basic }} clusters](#basic-clusters) have a default configuration that you cannot modify.
```

Comment:
Should we be consistent for using settings vs. configuration? I wonder if settings makes more sense to use across the board. I'm impartial though.

Reply:
That makes sense! I think "Settings" maybe, given that that is what we have in the UI. It may be "configuration" in the API — but I'm not really sure how much of that customers will see.

I may use the verb forms "set" and "configure" just because the language will get pretty tedious/hard to read otherwise. But, noun-wise, I'll keep to "Settings" if you're OK with that.

Reply:
I've removed the "configuration" mention as per this ^